modules/monitoring-configuring-the-cluster-monitoring-stack.adoc

```diff
@@ -50,7 +50,8 @@ data:
     *prometheusK8s*:
       *volumeClaimTemplate:
        spec:
-         storageClassName: gluster-block
+         storageClassName: fast
+         volumeMode: filesystem
```

Comment:
It is not necessary for
volumeMode: filesystem
we could remove it

Reply:
No, we should leave it. In cases where this is a default setting it won't change anything, but in cases where it is not a default setting it will point others to use the only supported mode.